modules/sandboxed-containers-deploy-node-check-feature-eligible-nodes.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * sandboxed_containers/deploying-sandboxed-container-workloads.adoc
+
+:_module-type: PROCEDURE
+[id="sandboxed-containers-deploy-node-check-feature_{context}"]
+= Deploying the check node feasibility feature on all eligible nodes
```

Comment:
The title doesn't reflect the following instructions. We really don't deploy a check node feasibility feature.
I suggest merging this section with the top one as mentioned in the following content - https://github.com/openshift/openshift-docs/pull/42831/files#r823533572

Reply:
Will do.